src/components/dashboard.js

```diff
@@ -1,7 +1,214 @@
+          <div class="top-bar mt-3 mb-4">
+            <nav class="top-nav">
+              <div class="input-group">
+                <input
+                  type="text"
+                  class="form-control"
+                  placeholder="Search dashboard"
+                  aria-label="Recipient's username"
+                  aria-describedby="button-addon2"
+                />
+                <div class="input-group-append">
+                  <button
+                    class="btn btn-outline-secondary bg-blue"
+                    type="button"
+                    id="button-addon2"
+                  >
+                    <img src="img/search.png" alt="" />
+                  </button>
+                </div>
+              </div>
+
+              <div class="user__details">
+                <div class="">
+                  <img src="img/notification.png" alt="notification-icon" />
+                </div>
+
+                <div class="display-profile">
+                  <div class="display-picture"></div>
+                  <p class="mt-2 ml-4">John Doe</p>
+                </div>
+
+                <div class="hamburger-menu">
+                  <i class="fas fa-bars fa-2x text-blue"></i>
+                </div>
+              </div>
+            </nav>
+          </div>
+
+          <div class="main-top">
+            <img class="ellipse_top img-fluid" src="img/Ellipse.png" />
+
+            <div class="row">
+              <div class="user__welcome col-lg-8 col-md-8 col-sm-8">
+                <p class="user__name">Hi, Barly Vallendito</p>
+                <p>
+                  Sinau Business Class Student is a shared for class to
+                  collaborate, communicate, manage information. Powerful
+                  Features And Integrations.
+                </p>
+                <small>*Business Class Start Early October 2019</small>
+              </div>
+
+              <div class="get__started col-lg-4 col-md-4 col-sm-8 d-flex align-items-end">
+                <button class="btn btn-block btn-primary mr-5">
+                  Get Started
+                </button>
+              </div>
+            </div>
+            {/* <!-- <img src="" class="background-vector img-fluid" alt="" srcset=""> --> */}
+            <span id="close__user__info">x</span>
+          </div>
+          <div class="cards">
+            <div class="card">
+              <div class="card-body mentor-card">
+                <div class=" card-image card-image-1">
+                  <img class="img-fluid" src="img/Group.png" />
+                </div>
+                <h6 class="card-subtitle">
+                  <span>1</span>
+                  <p>Approved Mentor</p>
+                </h6>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body resource-card">
+                <div class="card-image card-image-2">
+                  <img class="img-fluid" src="img/newspaper.png" />
```

Comment:
Codacy found an issue: [img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5730659)